### PR TITLE
[batch] introduce Support-Logs-Specs-and-Firewall fee

### PR DESCRIPTION
**Pull request by danking**
Fixes https://github.com/hail-is/hail/issues/13526

I verified I got the right number. I didn't import since I think we decided we shouldn't rely on the code base in the migrations.

```
In [1]: from hailtop.utils.rates import rate_instance_hour_to_fraction_msec, rate_cpu_hour_to_mcpu_msec
   ...: 
   ...: rate_cpu_hour_to_mcpu_msec(0.005) == 0.000000000001388888888888889
```

### CODE REVIEW

**Comment by danking** [comment]
@jigold not actually WIP, but I want control over when it merges. Ready for review.

File: batch/batch/cloud/gcp/instance_config.py [hunk 1]
```diff
@@ -151,6 +153,7 @@ def from_dict(data: dict) -> 'GCPSlimInstanceConfig':
                 GCPDynamicSizedDiskResource('disk/pd-ssd/1'),
                 GCPIPFeeResource('service-fee/1'),
                 GCPServiceFeeResource('ip-fee/1024/1'),
+                GCPSupportLogsSpecsAndFirewallFees('gcp-support-logs-specs-and-firewall-fees/1'),
```
Comment (jigold) [inline batch/batch/cloud/gcp/instance_config.py:156]: I don't think you need this as it's for backwards compatibility. See comment in #13539.

**Review by jigold — CHANGES_REQUESTED** [review]

File: batch/batch/cloud/gcp/instance_config.py (hunk 1 above)
Comment (danking) [reply batch/batch/cloud/gcp/instance_config.py:156]: Can you elaborate on what kind of backwards compatibility? Should we remove this branch entirely?

Comment (jigold) [reply batch/batch/cloud/gcp/instance_config.py:156]: I think we can. It was for when the `instance_config` field was not in the database for old entries, so their value is NULL. We knew what the config should be at the time though. The only reason we'd need this code path is if we wanted to compute resources for old instances. I don't know why we would want to do that.

Comment (danking) [reply batch/batch/cloud/gcp/instance_config.py:156]: https://github.com/hail-is/hail/pull/create?base=hail-is%3Amain&head=danking%3Aremove-unnecessary-backwards-compatibility-code&head_repo=hail

**Review by jigold — APPROVED** [review]

**Pull request merged**